gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-version=1.0.0
+version=1.1.0
```

Comment:
I went ahead and increased it to 2.0.0, just in case, there is the slight off chance that it might bite someone otherwise